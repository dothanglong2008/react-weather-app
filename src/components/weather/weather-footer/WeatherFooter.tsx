import styles from './WeatherFooter.module.scss';

function WeatherFooter() {
  return (
    <p className={styles.resource}>
      <span>API Source: </span>
      <a href="https://www.weatherapi.com/">weatherapi.com</a>
    </p>
  );
}

export default WeatherFooter;
