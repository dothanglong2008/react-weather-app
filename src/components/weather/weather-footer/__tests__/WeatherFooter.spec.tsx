import renderer from "react-test-renderer";
import WeatherFooter from "../WeatherFooter";

describe("Test WeatherFooter component", () => {
  it("renders WeatherFooter correctly", () => {
    const tree = renderer.create(<WeatherFooter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
