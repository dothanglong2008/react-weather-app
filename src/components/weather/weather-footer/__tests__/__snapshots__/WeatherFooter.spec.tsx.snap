// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test WeatherFooter component renders WeatherFooter correctly 1`] = `
<p
  className="resource"
>
  <span>
    API Source: 
  </span>
  <a
    href="https://www.weatherapi.com/"
  >
    weatherapi.com
  </a>
</p>
`;
